core/src/main/antlr4/com/databricks/labs/remorph/parsers/library/USQlCursors.g4

```diff
@@ -0,0 +1,158 @@
+// MIT License
```

Comment:
the code in this repo is under https://github.com/databrickslabs/remorph/blob/main/LICENSE, where this license is coming from? what is the original repository?

Reply:
https://github.com/jimidle/tsqlparser